nshlib/nsh_dbgcmds.c

```diff
@@ -473,7 +474,8 @@ int cmd_hexdump(FAR struct nsh_vtbl_s *vtbl, int argc, FAR char **argv)
             }
 #endif
 
-          snprintf(msg, sizeof(msg), "%s at %08x", argv[1], position);
+          snprintf(msg, sizeof(msg), "%s at %08zx", argv[1],
+                   (uintmax_t)position);
```

Comment:
should we use ssize_t?

Reply:
i changed it to use uintmax_t consistently.